packages/components/organisms/f-offers/package.json

```diff
@@ -61,6 +61,9 @@
     "js-cookie": "2.2.1",
     "@justeat/f-trak": "0.7.1",
     "@justeat/f-media-element": "0.3.0",
-    "@justeat/f-button": "1.7.0"
+    "@justeat/f-button": "1.7.0",
+    "@justeat/f-searchbox": "3.13.0",
```

Comment:
Is the addition of f-searchbox part of this PR, or related to a different one?

Reply:
its for a previous PR, I missed the package off